client/src/components/Search/Search.js

```diff
@@ -362,8 +377,9 @@ class Search extends Component {
 
     const searchText = searchData[name] || '';
     // Data to store in the redux state
+    console.log('overrides are ', overrides);
```

Comment:
remember to remove these before merging.

Reply:
Yeah, the build won't allow it anyway. Long way from merge. :-)